about.html

```diff
@@ -0,0 +1,112 @@
+        <a id="novaLink" href="./index.html">Nova</a>
+    </div>
+    <section id="about">
+        <article class="mainArtAbout">
+            <img class="greyFullImg" src="./Assets/images/greyPatful.jpg" alt="">
+            <h1 class="storyAboutH1">Nova Cosmetics</h1>
+            <h2 class="storyAboutH2">Our Story Started in 2020 when Pandamic spreaded</h2>
+
+            <p class="storyParagraphAbout1">At Nova Cosmetics, we believe in the power of natural beauty. Our mission is to create high-quality, natural products that enhance your natural beauty and make you feel confident and empowered. <br>We are dedicated to using only the finest, all-natural ingredients in our products. Our team of scientists and skincare</p>
+
+            <p class="storyParagraphAbout2">At Nova Cosmetics, we believe in the power of natural beauty. Our mission is to create high-quality, natural products that enhance your natural beauty and make you feel confident and empowered. <br>We are dedicated to using only the finest, all-natural ingredients in our products. Our team of scientists and skincare
+            experts work tirelessly to research and develop formulations that are not only effective, but also gentle and nourishing <br> for your skin.
+            We are committed to sustainability and ethical practices, and we source our ingredients from reputable, environmentally<br>
+            responsible suppliers. We never test on animals and all of our products are vegan and cruelty-free.
+            </p>
+            <div id="aboutContact">
+            <p>
+                Please contact us through our email address If you had any other questions or inquiries <br>
+                about Nova Cosmetics
+            </p>
+            <a href="demo@novact.com">demo@novact.com</a>
+            </div>
+        </article>
+        <article class="textAbout2">
+            <h2>Nova Cosmetics 2022</h2>
+            <p>
+                The logo of Nova Cosmetics was decided through the logo competition from 8, June to 7, July.
+            </p>
+            <div class="logoAbout">
+                <img class="logoAbout" src="./Assets/images/logo.png" alt="">
+            </div>
+        </article>
+    </section>
+    <section class="secSectionAbout">
+        <article>
+            <h2>See the past Nova Events</h2>
+            <hr>
+            <p>Take a look at the past Nova events which took place in Dubai and in Erbil</p>
+        </article>
+        <section class="imgTextabt">
+        <div class="eventPhotoswText">
+            <img id="imgevent" src="./Assets/images/event22.jpg" alt="eventPhoto">
+            <div class="textcentered">Nova Cosmetics in Dubai <br>2022</div>
+        </div>
+        <div class="eventPhotoswText">
+            <img id="imgevent" src="./Assets/images/Event_Photo.jpg" alt="eventPhoto">
+            <div class="textcentered">Nova Cosmetics in Dubai <br>2022</div>
+        </div>
+        </section>
```

Comment:
**OPTIONAL** - This section of your page overlaps on the navbar, please fix this to help improve the UI of your site 😄 
**OPTIONAL** - Also there are little differences in the design of the section, please remember to stick to the colors, texts, and background colors for the project. The images have a color overlay that is different from what you have used in your project. Please fix this 😄 

Your project:
<img width="960" alt="hwrhawrh" src="https://user-images.githubusercontent.com/73607512/210562984-5ecaf38c-59e1-47d5-995c-8a513fd8b481.PNG">

Design template:
<img width="482" alt="etaetjstj" src="https://user-images.githubusercontent.com/73607512/210563116-987691b0-4d48-409f-9632-2ebcc7aacae6.PNG">

Reply:
Thanks for mentioning that I fixed it